src/libvirt-common.js

```diff
@@ -44,7 +44,6 @@ import {
 import store from './store.js';
 import VMS_CONFIG from './config.js';
 
-const _ = cockpit.gettext;
```

Comment:
I don't understand how it can build in CI? That uses exactly the same eslint version.. do you have a local one which is prefered over the one from `npm install`?

Reply:
I thought so as well, but now I just double checked and this is not the case. Seems some issue with webpack5/eslint probably.

Running make on master now emits no warnigns/errros.
However, $ ./node_modules/eslint/bin/eslint.js /home/kkoukiou/repos/cockpit-machines/src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

Shows one warning.

I will check it